music_album.rb

```diff
@@ -0,0 +1,22 @@
+require_relative 'items'
+
+class MusicAlbum < Item
+  def initialize(on_spotify: false)
+    super
+    @on_spotify = on_spotify
+  end
+
+  def on_spotify?
+    # Should this be like: "return true if @on_spotify", or just this keyword is enough..?
+    @on_spotify
+  end
```

Comment:
For the MusicAlbum class, I think we only need `on_spotify ` as a property and the only method the class has is the override of `can_be_archived?`, so I think we could remove this method.
![image](https://user-images.githubusercontent.com/101157253/214335256-70b204c9-a369-4a58-892f-bf126edc11e7.png)

Reply:
you are right, i removed it!